pages/_app.js

```diff
@@ -1,77 +1,48 @@
-import '../styles/globals.css'
-import '../styles/globals.scss'
+import '../styles/globals.css';
+import '../styles/globals.scss';
 import React from 'react';
-import App from 'next/app'
-import Link from 'next/link'
+import App from 'next/app';
+import Link from 'next/link';
 import { useEffect } from 'react';
-import {serviceName} from '../helpers/constants';
+import Header from '../compoments/header';
 
-function MyApp({ Component, pageProps, err  }) {
+function MyApp({ Component, pageProps, err }) {
   const enableJavascript = () => {
-    window.GOVUKFrontend.initAll()
-  }
-  useEffect(enableJavascript, [])
+    window.GOVUKFrontend.initAll();
+  };
+  useEffect(enableJavascript, []);
 
-  return <>
-    <header className="govuk-header " role="banner" data-module="govuk-header">
-      <div className="govuk-header__container govuk-width-container">
-        <div className="govuk-header__logo">
-          {/* eslint-disable-next-line @next/next/no-html-link-for-pages */}
-          <a href="https://www.lincoln.gov.uk/" className="govuk-header__logotype" target="_blank" rel="noopener noreferrer"/>
```

Comment:
Should we be opening this in a new tab with `target="blank"` ? It's inconsistent from other services on NSDC. 

We should also add an aria-label like
`aria-label="Neward and Sheerwood Council Home page"` or something similar